lib/qubit.rb

```diff
@@ -0,0 +1,70 @@
+require "matrix"
+require_relative "vector_helper"
+require_relative "extensions/extensions"
+
+ZERO_PROB = "|0>".freeze
+ONE_PROB = "|1>".freeze
+
+# Qubit class is quantum bit abstraction. It contains 2 - dimensional vector
+# which represents probability of 0 and 1
+# Qubit can be measured or multiplicative
+class Qubit
+  attr_reader :vector
+  attr_accessor :entanglement
+
+  include VectorHelper
+
+  # Initialization by two probability of zero and one
+  # Probabilities should be normalized: |a| + |b| = 1
```

Comment:
there is a mistake
Probabilities should be normalized: |a|^2 + |b|^2 = 1

Reply:
fixed in 5ff6e41